examples/src/main/java/com/google/instrumentation/examples/trace/BasicTracing.java

```diff
@@ -1,35 +0,0 @@
-/*
- * Copyright 2017, Google Inc.
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *     http://www.apache.org/licenses/LICENSE-2.0
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.google.instrumentation.examples.trace;
-
-import com.google.instrumentation.trace.Span;
-import com.google.instrumentation.trace.Tracer;
-import com.google.instrumentation.trace.Tracing;
-
-/** Example showing how to create a {@link Span} and add annotations. */
-public final class BasicTracing {
```

Comment:
We do not compile or generate binary for these files and decided to remove them since they are not useful.